src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -99,12 +99,6 @@
 "ThreadingModel"="Both"
 @="Microsoft.VisualStudio.Editors.SettingsDesigner.PublicSettingsSingleFileGenerator"
 
-[$RootKey$\CLSID\{9AF66DCA-14CC-42C1-A52D-F403A4A43076}]
```

Comment:
Can we delete this one too http://index/?query=%60CLSID_CVersionPropPg2&rightProject=VS(Repo)&file=src%5Cvsproject%5Ccool%5Ccoolpkg%5Ccoolpkgguid.h&line=35?

Reply:
https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/223997?_a=overview